src/_locales/zh/messages.json

```diff
@@ -9,7 +9,7 @@
         "message": "現在發佈！"
     },
     "additional_post_box__encrypted_post_pre": {
-        "message": "來 #Maskbook 解密這個訊息！ $1",
```

Comment:
Does it change the template on the Facebook?

Reply:
yes...